src/team/components/Team.tsx

```diff
@@ -2,8 +2,9 @@ import React, { useState, useEffect } from 'react';
 import styled from 'styled-components';
 import isEqual from 'lodash.isequal';
 import { Scrollbars } from 'rc-scrollbars';
+import { useParams } from 'react-router-dom';
 import { Scrollers, BackgroundWrapper, BackgroundWhiteWrapper, flexCenter } from 'shared/components/';
-import Link from 'shared/components/Link';
+import { Link } from 'shared/components/Link';
 import goldLogo from '../../assets/images/gold_logo.png';
```

Comment:
i think this can work too -
import goldLogo from 'assets/images/gold_logo.png';